caffe2/ideep/operators/utility_ops.cc

```diff
@@ -1,5 +1,3 @@
-#include "caffe2/operators/utility_ops.h"
-#include "caffe2/core/operator.h"
```

Comment:
I don't see why this change should go in this patch

Reply:
just remove useless code.
if not good, I'll delete this change.